src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.250.0.mdx

```diff
@@ -0,0 +1,31 @@
+---
+subject: Browser agent
+releaseDate: "2024-01-09"
+version: 1.250.0
+features: ["Add a flag to note that a payload occurred during a replay","Session Replay Dynamic Loading"]
+bugs: ["TraceId generation regression (non-random trace ids)"]
+security: []
+---
+
+## v1.250.0
+
+### Features
+
+#### Add a flag to note that a payload occurred during a replay
+Added a flag to mark harvests as having occurred during an active session replay. This will aid cross-feature querying in NR1.
+
+#### Session Replay Dynamic Loading
+Allow Session Replay modules to load earlier in the page lifecycle if a replay is ongoing
+
+### Bug fixes
+
+#### TraceId generation regression (non-random trace ids)
+Fixes a regression which caused all trace ids generated to end in a 0.
+
+## Support statement:
+
+New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Older releases will no longer be supported when they reach [end-of-life](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/). Release dates are reflective of the original publish date of the agent version.
+
+New Browser Agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```

Comment:
```suggestion
New Relic recommends that you upgrade the agent regularly to ensure that you're getting the latest features and performance benefits. Older releases will no longer be supported when they reach [end-of-life](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/). Release dates are reflective of the original publish date of the agent version.

New browser agent releases are rolled out to customers in small stages over a period of time. Because of this, the date the release becomes accessible to your account may not match the original publish date. Please see this [status dashboard](https://newrelic.github.io/newrelic-browser-agent-release/) for more information.
```